worlds/subnautica/docs/setup_en.md

```diff
@@ -19,7 +19,7 @@
 Use the connect form in Subnautica's main menu to enter your connection information to connect to an Archipelago multiworld.
 Connection information consists of:
  - Host: the full url that you're trying to connect to, such as `archipelago.gg:38281`.
- - PlayerName: your name in the multiworld. Can also be called "slot name" and is the name you entered when creating your settings.
+ - PlayerName: your name in the multiworld. Can also be called "slot name" and is the name you entered when creating your options.
```

Comment:
```suggestion
 - PlayerName: your name in the multiworld. Can also be called "slot name" and is the name you entered when you chose your options.
```

Reply:
```suggestion
 - PlayerName: your name in the multiworld. Can also be called "slot name" and is the name you entered when choosing your options.
```

Reply:
Either way should work. It's up to Berserker here, anyway

Reply:
yep